src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -107,6 +110,24 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [ConditionalTheory(nameof(IsWindows10Version1607)), MemberData(nameof(Http2Servers))]
```

Comment:
Could we move this then to the System.Net.Http.Functional.Tests project?  We can then subsequently change the condition to one that will also help test the unix side.